quilt/tools/build.py

```diff
@@ -312,14 +312,21 @@ def _ignored_name(name):
             name == outfilename
         )
 
+    def _is_empty_dir(dir_path):
+        return os.path.isdir(dir_path) and not os.listdir(dir_path)
```

Comment:
Tthat's not good enough: the directory could contain files starting with `.`, `build.yml`, `quilt_packages`, other empty directories - which all get ignored. You should handle this at a higher level - maybe even support empty directories and just allow empty groups.

Reply:
Another to support empty groups: I have the following build file with commented-out lines, and it fails:
```
contents:
  README:
    file: README.md
  copy_uniques_level0_unclustered_sample_vectorized:
    file: proc_17_108_copy_uniques_level0_unclustered_sample_vectorized.csv
  unique_comments_text_dupe_count:
#    part1:
#      file: proc_17_108_unique_comments_text_dupe_count.csv
#      nrows: 1500000
#    part2:
#      file: proc_17_108_unique_comments_text_dupe_count.csv
#      skiprows: 1500001
```